lldb/source/Plugins/SymbolFile/DWARF/SymbolFileDWARF.cpp

```diff
@@ -982,7 +982,7 @@ lldb::LanguageType SymbolFileDWARF::ParseLanguage(CompileUnit &comp_unit) {
     return eLanguageTypeUnknown;
 }
 
-XcodeSDK SymbolFileDWARF::ParseXcodeSDK(CompileUnit &comp_unit) {
+XcodeSDKPath SymbolFileDWARF::ParseXcodeSDK(CompileUnit &comp_unit) {
```

Comment:
Thinking out aloud: I was just going to write: "wouldn't it be better if this function returned an abstract SDK specification?" but then I realized that this is the point where we can read an absolute oath to the sysroot out of DWARF, so it needs to be am absolute path.